scripts/declarations/utils/index.js

```diff
@@ -33,48 +36,55 @@ export default class DeclarationUtils {
     return modifiedServicesIds;
   }
 
-  async getModifiedServiceTermsTypes() {
-    const { modifiedFilePaths, modifiedServicesIds } = await this.getModifiedData();
+  async getModifiedServicesTermsTypes() {
```

Comment:
If, as I understand from the tests, this actually returns `{ services: ['serviceId' ], servicesTermsTypes: { serviceId: [ termsTypes] } }`, it should be named 
```suggestion
  async getModifiedServicesAndTermsTypes() {
```

(but also, why should it duplicate information that can be obtained with `Object.keys`??!)

Reply:
I not changed the previous API